src/entities/commit/enum/commitFormatStyleEnum.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * 가장 흔한 커밋 메세지 형식의 종류를 정의합니다.
+ */
+const COMMIT_FORMAT_STYLE = {
+  prefix: {
+    type: "prefix-style",
+    splitWith: ":",
+  },
+  simpleText: {
+    type: "simple-text-style",
+    splitWith: " ",
+  },
+  templateBased: {
+    type: "template-based",
+    splitWith: "]",
+  },
+  sentenceCase: {
+    type: "sentence-case",
+    splitWith: " ",
+  },
```

Comment:
sentenceCase는 simpleText와 splitWith가 겹쳐서 아직은 차별되지 않으니 지금은 지워주는 것이 더 좋을 것 같다는 생각이 들었습니다!
여러분의 의견은 어떠신가요~
```suggestion
  sentenceCase: {
    type: "sentence-case",
    splitWith: " ",
  },
```